dask/dataframe/core.py

```diff
@@ -2568,6 +2572,10 @@ def clip_lower(self, threshold):
     def clip_upper(self, threshold):
         return self.map_partitions(M.clip_upper, threshold=threshold)
 
+    @derived_from(pd.DataFrame)
+    def squeeze(self, axis=None):
+        return self.map_partitions(M.squeeze, axis=axis)
```

Comment:
It's not sure to me what should happen if `axis=0`.  I don't think we want to push this down to the partitions.  For example if our dataframe has two partitions each with one row we would not want to turn that into two pandas Series objects.